cookbook/deployment/fortrabbit.rst

```diff
@@ -197,8 +197,8 @@ Deploying to fortrabbit
 It is assumed that your codebase is under version-control with Git and dependencies
 are managed with Composer (locally).
 
-Every time you push to fortrabbit composer install runs before your code gets 
-deployed. To finetune the deployment behavior put a `fortrabbit.yml`_. deployment 
+Every time you push to fortrabbit composer install runs before your code gets
+deployed. To finetune the deployment behavior put a `fortrabbit.yml`_. deployment
```

Comment:
I think this is my "remove trailing spaces on save", but not sure. @xabbuh should I revert this?